flake.nix

```diff
@@ -91,18 +91,23 @@
               runHook preInstall
 
               install -Dm755 $GOPATH/bin/numen -t $out/bin
+              install -Dm755 $src/numenc -t $out/bin
               export NUMEN_SKIP_BINARY=yes
               export NUMEN_SKIP_CHECKS=yes
               export NUMEN_DEFAULT_PHRASES_DIR=/etc/numen/phrases
               export NUMEN_SCRIPTS_DIR=/etc/numen/scripts
               ./install-numen.sh $out $out/bin
+              rm -r $out/nix
```

Comment:
Looking at `install-numen.sh`, it seems like the second argument is only used for the installation of numenc. It should be enough to change $out/bin to just /bin in the line above. Then we also wouldn't have to clean up the extraneous file here.